pkg/appstore/app.go

```diff
@@ -26,3 +31,21 @@ func (a App) MarshalZerologObject(event *zerolog.Event) {
 		Str("version", a.Version).
 		Float64("price", a.Price)
 }
+
+func (a App) GetIPAName() string {
```

Comment:
The function is only used within the `appstore` so I think we can keep it internal. Additionally, no need to include `get` in the getter function.

```suggestion
func (a App) fileName() string {
```

Reply:
There were a few reasons I put that there:
1. The function solely uses app's properties to build the return value, it felt weird to have appstore do it
2. Testing is much easier, if I were to put this in appstore it would need to be a public method so the tests can access it

Happy to refactor as you wish though if that's what you prefer.

Reply:
I'm not sure what you are referring to here. The function is already in the `appstore` package. I am suggesting renaming it to a lowercase function so that it becomes internal (as it is only used from within the `appstore` package at the moment.